### PR TITLE
I created a method in the python file to update db with new user and …

### PR DESCRIPTION
**Pull request by jishaanm**
…updated js file accordingly

### CODE REVIEW

File: application.py [hunk 1]
```diff
@@ -158,7 +158,25 @@ def check_login_creds():
...
+
+    user_id = User.query.filter_by(email = email).first().user_ID
+    output = {'response':user_id}
+
+    return jsonify(output)
+
 # Run server
 if __name__ == '__main__':
-    application.run(host='0.0.0.0') 
-    # application.run(debug=True)
```
Comment (andreimastin) [inline application.py]: Fixed back to General host

**Review by andreimastin — CHANGES_REQUESTED** [review]
Looks good make one change to server host

File: application.py (hunk 1 above)
Comment (jishaanm) [reply application.py]: Thanks I will update it accordingly

**Review by andreimastin — APPROVED** [review]

**Comment by jishaanm** [comment]
Reviewed and updated the conflicts accordingly

**Pull request merged**